src/SFA.DAS.AdminService.Settings/WebConfiguration.cs

```diff
@@ -5,10 +5,6 @@ namespace SFA.DAS.AdminService.Settings
 {
     public class WebConfiguration : IWebConfiguration
     {
-        public AuthSettings Authentication { get; set; }
-
-        [JsonRequired] public ApiAuthentication ApiAuthentication { get; set; }
-
         [JsonRequired] public AzureApiAuthentication AzureApiAuthentication { get; set; }
 
         [JsonRequired] public ClientApiAuthentication ClientApiAuthentication { get; set; }
```

Comment:
There are several used of ClientApiAuthentication - this one is for the Admin Web to connect to the Assessor Api - however it should be called EpaoApiAuthentication - as it is the equivalent of the RoatpApiAuthentication for Epao - and also because for some reason Roatp uses the term Assessor to actually mean the Admin service...